mythril/analysis/call_helpers.py

```diff
@@ -0,0 +1,49 @@
+from mythril.analysis.ops import VarType, Call, get_variable
+
+
+def get_call_from_state(state):
```

Comment:
Thanks for this helper. :smile: Can you add some type annotations?

Reply:
Done